src/main/java/mlib/tool/MVHttpClient.java

```diff
@@ -1,10 +1,10 @@
-package mSearch.tool;
+package mlib.tool;
+
+import java.util.concurrent.TimeUnit;
 
 import okhttp3.ConnectionPool;
 import okhttp3.OkHttpClient;
 
-import java.util.concurrent.TimeUnit;
-
 public class MVHttpClient {
     private final static MVHttpClient ourInstance = new MVHttpClient();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)